index.html

```diff
@@ -470,7 +470,7 @@ <h3>Prohibited States and Properties</h3>
 			</section>
 		<section id="mustContain">
 			<h3>Required Owned Elements</h3>
-			<p>Any <a>element</a> that will be <a>owned</a> by the element with this <a>role</a>. For example, an element with the role <rref>list</rref> will own at least one element with the role <rref>listitem</rref>.</p>
+			<p>The <a>role</a> required for all <a>owned</a> children. For example, an element with the role <rref>list</rref> will own at least one element with the role <rref>listitem</rref>. Authors MUST ensure all <a>owned</a> child nodes, and all elements referenced by <pref>aria-owns</pref> from the active element, have at least one of the roles specified in the <em>required owned element</em> for the role of the active element.</p>
```

Comment:
```suggestion
			<p>The <a>role</a> or roles required for all <a>owned elements</a> of an element with this role. For example, an element with the role <rref>list</rref> will own at least one element with the role <rref>listitem</rref>. Authors MUST ensure that all <a>owned elements</a> have one of the roles specified in the <em>required owned element</em> for this role.</p>
```

Notes:
Other definitions in [5.2 Characteristics of Roles](https://w3c.github.io/aria/#Properties) 
use "this role", "the role" or "a role" to refer to the thing the definition applies to.

We can take advantage of the definition of "owned element" (so that we don't need as many words):
> [Owned Element](https://w3c.github.io/aria/#dfn-owned-element)
> An 'owned element' is any DOM descendant of the element, any element specified as a child via aria-owns, or any DOM descendant of the owned child.

Here's what the above suggested change looks like in prose:
> The role or roles required for all owned elements of an element with this role. For example, an element with the role list will own at least one child element with the role listitem. Authors MUST ensure that all owned elements have one of the roles specified in the required owned elements for this role.

Reply:
Not using this suggestion.